dts/arm/nxp/nxp_rt.dtsi

```diff
@@ -621,21 +621,29 @@
 		};
 
 		usdhc1: usdhc@402c0000 {
-			compatible = "nxp,imx-usdhc";
```

Comment:
usdhc is the IP name listed in our reference manual, is it possible to keep to usdhc?

Reply:
That name would conflict with NXP's current USDHC driver. Once all platforms using that driver transition to the new SDHC driver, we could use the `nxp,imx-usdhc` binding for this new SDHC driver, but for the time being this name is required to allow both drivers to coexist.

Reply:
Is USDHC NXP's own design or can it be found in the SoC of other vendors?

Reply:
USDHC is NXP's own design as far as I am aware. SD host controllers do have a specification they conform to, but I opted not to implement a generic SD host controller driver with vendor specific overrides (the approach Linux takes) for a few reasons:

1. Zephyr allows vendor HALs, so most vendor drivers can be implemented as a shim and rely on tested HAL code to manage the SDHC IP
2. Code size should be reduced by skipping the abstraction of a common SD host controller driver with vendor overrides
3. If NXP's USDHC is anything to go by, SD host controllers often deviate from the SD specification, so a generic layer may not be very useful.

Reply:
The old and new SDHC drivers won't coexist in the same application though; there are Kconfig symbols to enable one or the other. I don't think this compatible should change.